webapp/src/js/controllers/about.js

```diff
@@ -53,13 +51,22 @@ angular.module('inboxControllers').controller('AboutCtrl',
       })
       .then(rev => ctrl.remoteRev = rev);
 
+    ctrl.knockCount = 0;
+    ctrl.secretDoor = () => {
+      if (ctrl.doorTimeout) {
+        $timeout.cancel(ctrl.doorTimeout);
+      }
+      if (ctrl.knockCount++ >= 5) {
+        $state.go('testing');
```

Comment:
just wondering if there is a way back to the main app after entering the testing state.  it requires browser back button? i didn't actually run this code

Reply:
You get all the usual navigation buttons you had before so you can easily switch to another tab:

![Screenshot from 2019-11-20 09-12-20](https://user-images.githubusercontent.com/2064938/69182459-edfbc200-0b75-11ea-8ca2-b823dd0bdf45.png)

The back button also works.